charts/managed-serviceaccount/templates/clustermanagementaddon.yaml

```diff
@@ -6,3 +6,6 @@ spec:
   addOnMeta:
     displayName: managed-serviceaccount
     description: managed-serviceaccount
+  supportedConfigs:
```

Comment:
do we need to add this by default? or we have a variable on chart to decide.

Reply:
yeah, put a variable on chart may be a better way

Reply:
add `enableAddOnDeploymentConfig ` to control this